### PR TITLE
fixes fucking slaughter demons spawning randomly an hour in

### PR DESCRIPTION
**Pull request by ghost**
it's two hours not one hour jfc we don't need random fucking demons showing up it was supposed to stay at 2hrs and whoever changed this is a fucking boobie dick shit boob
#### Changelog

:cl:
rscadd: FUCKIGN SLAUGHTER DEMONS NO LONGER RANDOMLY SPAWN AN HOUR IN
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
Rounds can't last 2 hours, most rounds don't even last 1. :-1:

**Comment by Super3222** [comment]
there's a changelog type called "tweak"

**Comment by Time-Green** [comment]
At least update the comment

**Comment by Super3222** [comment]
Slaughter Demons are overpowered wizard mobs. I don't want to see one randomly spawn in on a lucky shift that's gone on for 1 hour. If they're that successful you should give them some time.

File: code/game/gamemodes/miniantags/slaughter/slaughterevent.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	typepath = /datum/round_event/ghost_role/slaughter
 	weight = 1 //Very rare
 	max_occurrences = 1
-	earliest_start = 36000 //1 hour
+	earliest_start = 72000//1 hour
```
Comment (Time-Green) [inline code/game/gamemodes/miniantags/slaughter/slaughterevent.dm]: UPDATE THE COMMENT AHHHHH
//2 hours

**Pull request merged**